pkg/user/interactors/create_token_test.go

```diff
@@ -27,6 +29,8 @@ var ctCases = []CreateTokenTestCase{
 		"token successfully created",
 		"user@user.com",
 		"123456789abcdef",
+		"user",
+		0,
```

Comment:
Please add tests changing theses properties.

Reply:
Just completing this request if I wasn't clear. 
Add tests using empty string and `app` too.
Can add an property `functionInteractorMockcalled string` to check if the mock called the correct method

Reply:
Done!